packages/yourloops/pages/login/login.tsx

```diff
@@ -89,16 +89,13 @@ const loginStyle = makeStyles((theme: Theme) => {
 /**
  * Login page
  */
-// eslint-disable-next-line @typescript-eslint/ban-types
 function Login(): JSX.Element {
   const { t } = useTranslation("yourloops");
-  const historyHook = useHistory<{ from?: { pathname?: string; }; }>();
```

Comment:
removing this line and the line 101 introduce a minor regression. we could not redirect to origin url before login (ylp-676). However we could address this later, it is not a show stopper

Reply:
Good catch, I will fix it